cnf-certification-test/suite_test.go

```diff
@@ -131,6 +131,27 @@ func TestTest(t *testing.T) {
 
 	// Set up logging params for logrus
 	loghelper.SetLogFormat()
+
+	// Delete execution log (if any) prior to run
+	logFile := "tnf-execution.log"
+	_, fileErr := os.Stat(logFile)
+	if os.IsExist(fileErr) {
+		e := os.Remove(logFile)
+		if e != nil {
+			panic(e)
+		}
+	}
+
+	// Setup logging to file
+	f, err := os.OpenFile(logFile, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0o644)
+	if err != nil {
+		log.Errorf("failed to create log file: %v", err)
+		panic(err)
+	}
+	defer f.Close()
+	log.SetOutput(f)
```

Comment:
Isn't that removing the console logs and only leaving Ginkgo logs? It would be nice to preserve the console logs at info level and have more detailed file logs at debug or lower level. Logrus supports having several outputs natively with hooks. see https://github.com/sirupsen/logrus/tree/master/hooks/writer

Reply:
Whoops yes, adjusted to have the hooks like you suggested.

Reply:
I would use the hooks to write all the log levels to file and leave the default mechanism we have now for console, something like:
https://github.com/sebrandon1/cnf-certification-test/pull/109

Reply:
Merged.